draft/ZEP0003.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: ZEP0000
+description: Variable chunk sizes
+parent: template
+nav_order: 1
+---
+
+# ZEP 3 — Variable chunking
+
+---
+
+Authors:
+* Martin Durant ([@martindurant](https://github.com/martindurant)), Anaconda, Inc.
+
+Status: Draft
+
+Type: Specification
+
+Created: 2022-10-17
+
+## Abstract
+
+To allow the chunks of a zarr array to be hyperrectangles rather than hypercubes, with the chunk
```

Comment:
I think the standard term for this is a "rectilinear grid", while the existing spec allows only a "regular grid".
https://en.wikipedia.org/wiki/Regular_grid

Hypercube grid cells would mean a chunk shape of e.g. 4 x 4 x 4; the existing spec already allows hyperrectangles, e.g. 4 x 5 x 6.

Reply:
OK, makes sense, I'll change the term.